components/restapi/core/component.go

```diff
@@ -85,31 +87,29 @@ const (
 	// GET returns the output IDs of all UTXO changes.
 	RouteCommitmentByIndexUTXOChanges = "/commitments/by-index/:" + restapipkg.ParameterSlotIndex + "/utxo-changes"
 
-	// RouteAccountsByAcciuntID is the route for getting an account by its accountID.
-	// GET returns the account details.
-	RouteAccountsByAcciuntID = "/accounts/:" + restapipkg.ParameterAccountID
+	// RouteBlockIssuanceCredits is the route for getting block issuance credits balance for an account.
+	// GET returns the block issuance credits balance.
+	RouteBlockIssuanceCredits = "/accounts/:" + restapipkg.ParameterAccountID
```

Comment:
Why do we have 2 separate routes? I thought we agreed on adding both BIC and the congestion stuff to one single endpoint, since a wallet would query both anyway

Reply:
I added separate BIC route again, because while implementing I realized we do not provide any endpoint to query BIC for a chosen slot. 
In BlockIssuuance enpoint we provide all details related to the issuance of the block, so the BIC should be provided for `slotIndex-MCA` only, cause this makes sense in the context of issuance, you check if balance is >0. 
So simple request about out latest balance or balance for specific slot  would be too confusing, and hidden with this blockissuance endpoint bc this `-MCA`.

Reply:
I would say we don't allow to specify the slotIndex for the congestion endpoint at all, and we remove the BIC endpoint because it might be confusing to show a positive BIC balance but the account could still have a negative BIC balance at `latestSlot-MCA`.

Reply:
We removed the `RouteBlockIssuanceCredits` endpoint.